### PR TITLE
Organize payments vertically

### PR DESCRIPTION
**Pull request by vincanger**
Implements a part of https://github.com/wasp-lang/open-saas/issues/149.

- [x] update template
- [x] update docs
- [x] update app_diff -- (didn't change anything in the app_diff)

### CODE REVIEW

**Review by Martinsos — APPROVED** [review]
Left some tiny comments but looks good to me! Nothing else we can pull from some other parts of the code? Some operations, or react compnoent, that would be better suited to be here? If not, great, let's merge! I approved.

**Comment by Martinsos** [comment]
@vinny I just noticed you haven't moved CHeckout Page, how is that? Is that on purpose?

File: opensaas-sh/blog/src/content/docs/start/guided-tour.md [hunk 1]
```diff
@@ -61,16 +61,16 @@ If you are using a version of the OpenSaaS template with Wasp `v0.11.x` or below
 ├── src/                   # Your code goes here.
 │   ├── client/            # Your client code (React) goes here.
 │   ├── server/            # Your server code (NodeJS) goes here.
-│   ├── shared/            # Your shared (runtime independent) code goes here.
 │   ├── auth/              # All auth-related pages/components and logic.
 │   ├── file-upload/       # Logic for uploading files to S3.
-│   └── .waspignore
```
[the remark this answers is not in the document]
Comment (vincanger) [reply opensaas-sh/blog/src/content/docs/start/guided-tour.md:67]: waspignore (_Was Pig Nore_) is in the main `src` dir, but I didn't think it was important enough to highlight in the docs here. WDYT?

File: opensaas-sh/blog/src/content/docs/start/guided-tour.md [hunk 2]
```diff
@@ -61,16 +61,16 @@ If you are using a version of the OpenSaaS template with Wasp `v0.11.x` or below
 ├── src/                   # Your code goes here.
 │   ├── client/            # Your client code (React) goes here.
 │   ├── server/            # Your server code (NodeJS) goes here.
-│   ├── shared/            # Your shared (runtime independent) code goes here.
```
[the remark this answers is not in the document]
Comment (vincanger) [reply opensaas-sh/blog/src/content/docs/start/guided-tour.md:64]: nope. nada. The leftovers got moved to `common` I believe

File: template/app/main.wasp [hunk 3]
```diff
@@ -411,6 +382,37 @@ job dailyStatsJob {
...
+  authRequired: true,
+  component: import Checkout from "@src/client/app/CheckoutPage"
+}
+
+action generateStripeCheckoutSession {
+  fn: import { generateStripeCheckoutSession } from "@src/payment/operations",
+  entities: [User]
+}
+
+/*
```
[the remark this answers is not in the document]
Comment (vincanger) [reply template/app/main.wasp:402]: I'd say we scrap them. The docs do a pretty good job of discussing this I think. I will add links to the relevant Wasp docs in the guided tour for each section of the Wasp file.

**Comment by vincanger** [comment]
@Martinsos whoops, forgot to move the CheckoutPage. Thanks for the heads up!

**Pull request merged**